src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -27,16 +27,17 @@ class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter) extends
 
   private val subscribedEventTypes = request.getParameterMap.getOrDefault("event_type", Array.empty).toSet
```

Comment:
Shouldn't `subscribedTaskIds` be introduced similarly to `subscribedEventTypes`?

Reply:
Done.